### ISSUE TITLE
Contain all content in a given page within landmarks

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#859]
**Describe the bug**
Contain all content in a given page within landmarks
	
**To Reproduce**
1. https://openfl.readthedocs.io/en/latest/index.html
2. Examine the code, and/or run the browser plugin for aXe Devtools

notice:  <footer> and versions expendable section in sidebar, are not within landmarks on the page

**Expected behavior**
<footer> should be part of a footer landmark and expandable section of sidebar should be within the <nav> element for the sidebar

**Screenshots**

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.

**Comment by fstrr** [comment]
This is an issue with the [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).

**Comment by theakshaypant** [comment]
Closing. As mentioned by @fstrr, this is an issue with [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).
